Tests/Unit-Tests/NewRelicAgentTests/Uncategorized/NRLoggerTests.m

```diff
@@ -192,12 +191,10 @@ - (void) testRemoteLogLevels {
         }
     }];
 
-    NSString *path = [NRLogger logFilePath];
-    NSData* logData = [NSData dataWithContentsOfFile:path];
-
+    NSError* error;
+    NSData* logData = [NRLogger logFileData:&error];
```

Comment:
Shall the localizedDescription of the error be printed if its non null after this operation?